Xamarin.PropertyEditing.Mac/Controls/BaseRectangleEditorControl.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections;
+using System.Diagnostics;
+using System.Drawing;
+using AppKit;
+using CoreGraphics;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal abstract class BaseRectangleEditorControl<T> : PropertyEditorControl
+		where T : struct
+	{
+		internal NSTextView XLabel { get; set; }
```

Comment:
I thought we weren't going to make these internal anymore? They should be protected and private set, nothing outside the controls should need to access them.